tests/rptest/scale_tests/many_clients_test.py

```diff
@@ -149,6 +154,19 @@ def test_many_clients(self, compacted):
             # Clients have to do the compression work on these larger messages,
             # so curb our expectations about how many we may run concurrently.
             producer_count = producer_count // 10
+        else:
+            producer_kwargs['min_record_size'] = 0
+            producer_kwargs['max_record_size'] = 16384
+
+        mean_msg_size = producer_kwargs['min_record_size'] + (
+            producer_kwargs['max_record_size'] -
+            producer_kwargs['min_record_size']) // 2
+        msg_rate = (target_throughput_mb_s * 1024 * 1024) // mean_msg_size
+        messages_per_sec_per_producer = msg_rate // self.PRODUCER_COUNT
+
+        # If this fails, the test was altered to have an impractical ratio of
+        # producers to traffic rate.
+        assert messages_per_sec_per_producer > 0, "Bad sizing params, need at least 1 MPS"
 
         producer = ProducerSwarm(self.test_context,
```

Comment:
Did you mean to also update `producer_kwargs` to include `records_per_second_per_producer` too in this commit?

Reply:
Fixed